packages/liveblocks-core/src/client.ts

```diff
@@ -657,9 +712,17 @@ export function createClient<U extends BaseUserMeta = DU>(
 
       ...httpClientLike,
 
+      // Advanced resolvers APIs
+      resolvers: {
+        invalidateUsers: invalidateResolvedUsers,
+        invalidateRoomsInfo: invalidateResolvedRoomsInfo,
+        invalidateMentionSuggestions: invalidateResolvedMentionSuggestions,
+      },
+
       // Internal
       [kInternal]: {
         currentUserIdStore,
+        mentionSuggestionsCache,
```

Comment:
Is this only exported to access it in unit tests?

Reply:
No it's used by `useMentionSuggestions` in `liveblocks-react` but it needs to live here at the client level in order to be accessible by `client.resolvers.invalidateMentionSuggestions`.